### DISCUSSION TITLE
umi3 的路由配置文档竟然是issue里找到的，文档真是太不友好了。

### DISCUSSION THREAD

**Discussion by ZangTengfei** [umijs/umi#6282] [General]
文档不好好写是诚心不想让人用？花了几个小时来找文档简直无语。

**Comment by zhangsai521314** [comment]
喷文档的都值得点赞

**Comment by leftstick** [comment]
哪里？我看看能否优化

**Reply by zhangsai521314** [reply]
大佬们写开源项目辛苦了，您们付出很多。作为一个刚接触的新手有点建议：

![image](https://user-images.githubusercontent.com/14134256/125382704-be43a280-e3c8-11eb-881d-34683259f71c.png)
这里没有给出/config文件内容的示列。
![image](https://user-images.githubusercontent.com/14134256/125382771-df0bf800-e3c8-11eb-8245-adaea54063a7.png)
没有给出在使用umi创建的项目使用@umijs/plugin-qiankun，使用此种方式应该如何使用qiankun的start
![image](https://user-images.githubusercontent.com/14134256/125383068-648fa800-e3c9-11eb-975d-ae1c2042fe66.png)
没有说明此种方式如果在config中删除qiankun-master-apps属性的时候在菜单里是出不来的，把qiankun-master-apps属性的apps数组赋值为[]是可以出来了
![image](https://user-images.githubusercontent.com/14134256/125383149-8b4dde80-e3c9-11eb-8daf-18874ed637d2.png)
此例子在整个umi里，并且使用了lerna管理整个项目，像我这种小白真的是苦苦的运行不起来

另外希望给出一个umi3.5+react pro v5+@umijs/plugin-qiankun当做主子应用的示列（因为我没弄起来），附问题链接：https://github.com/umijs/umi/issues/6964

 @没找到怎么输入表情。。。。。
